pkg/beyla/os_test.go

```diff
@@ -44,3 +49,109 @@ func TestCheckOSSupport_Unsupported(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Another nitpick. Golang's `assert.Equal` or `require.Equal` received the expected value as first argument, and the actual value as second argument.

This is a minor issue until tests fail and then the failing message might be a bit confusing.